src/manual/coupon-details.md

```diff
@@ -9,58 +9,69 @@ Label (required)
 : An internal label which helps you to identify the coupon and which can be used for searching in the administration interface.
 
 Provider (required)
-: This is the last part of the coupon provider class name, e.g. "FixedRebate" for the "\Aimeos\MShop\Coupon\Provider\FixedRebate". The name of the provider is **case sensitive**, so "fixedrebate" is not the same as "FixedRebate"! You will get a list of available providers when clicking on the input field. Each coupon provider can be enhanced by one or more decorators that must be added to the provider name and separated by a comma. You view the list of available decorators when clicking on the "+" symbol right of the input field. By clicking on the decorator name it will be added after the provider name or previously added decorators. For a detailed description of the available coupon providers and decorators that are part of the Aimeos core, please have a look at the [coupon overview](coupons.md) and the page for the [coupon decorators](coupon-decorators.md).
+: This is the last part of the coupon provider class name, e.g. "FixedRebate" for the "\Aimeos\MShop\Coupon\Provider\FixedRebate". The name of the provider is **case sensitive**, so "fixedrebate" is not the same as "FixedRebate"!    
+A list of available providers is shown when you click into the input field. Selecting a provider adds it to the input field. Furthermore, each provider can be enhanced by one or more decorators, which can be choosen from the drop-down list that appears when the "+" symbol next to the input field is clicked. Selecting a decorator adds it sequentially to the provider or any other previously added decorator(s).    
```

Comment:
Is that line added to the definition before or does the newline cause this as rendered as a new paragraph?

Reply:
This line belongs to the definition before. It is only a line break. Since this paragraph contains a lot of dense info, it helps to group them into their separate blocks: 
- Name (which is anyway too much of developer slang and belongs to the "developer" section instead of the user manual; should be shortend to "name of the Provider, e.g. 'FixedRebate' (case-sensitive). "
- Input handling
- Further info
Shortened in the next PR.